### PR TITLE
ENG-284 ENG-296 - Onboarding improvements and cheatsheet styling

### PR DESCRIPTION
**Pull request by floreks**
## Summary

- Updated the onboarding experience
  - Hide the sidebar when user is ONBOARDING for the first time (status `NEW`)
  - Hide demo and show cheatsheet after onboarding is completed or skipped
- Restored the `CLI Cheatsheet` button
- Update cheatsheet styling as per the design (https://www.figma.com/file/jbylrXNxx5WCsIhjb8GZYu/Cloud-Shell-(non-demo)?node-id=3%3A5685)

**NOTE**: Don't mind the old back icon in the cheatsheet commands in the video. I have already updated that.

Zrzut ekranu z 2022-07-08 12-16-05

### Questions
1. Do we want to remove the cluster name as in the design?
2. Should the cheatsheet be always open or closed when entering the shell view?
3. Do we want to force lock user inside the `/shell` view (force redirect) during onboarding? Right now it is possible to simply navigate to the `/` and see the full app with the sidebar.

### Recording 
[Nagranie ekranu z 08.07.2022 11:49:52.webm](https://user-images.githubusercontent.com/2285385/177969365-014e5347-2b42-405c-8bec-4eb228d4e5d7.webm)

## Test Plan

Locally

## Checklist

- [x] If required, I have updated the Plural documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] I have added a meaningful title and summary to convey the impact of this PR to a user.
- [x] I have added relevant labels to this PR to help with categorization for release notes.

### CODE REVIEW

File: www/package.json
Comment (maciaszczykm) [inline www/package.json:77]: Remove `^`.

File: www/src/components/shell/TerminalSidebar.js
Comment (dogmar) [inline www/src/components/shell/TerminalSidebar.js:557]: This this should be "topsort **them**,"

File: www/src/components/shell/Terminal.js
Comment (dogmar) [inline www/src/components/shell/Terminal.js:6]: I think we should start importing `Button` from 'plural-sh-design-system' from now on since that version has added functionality over the default honorable one. Doesn't _really_ matter here, but may as well be consistent in case it ever diverges from the honorable one.

File: www/src/components/shell/TerminalSidebar.js
Comment (dogmar) [inline www/src/components/shell/TerminalSidebar.js:506]: I think the text should be included in this button, and button should align with text below.
cheatsheet-markup
.

**Review by dogmar — COMMENTED** [review]
Should we default to opening the Commands tab in the cheatsheet for now since there's only one tab?

File: www/src/components/shell/TerminalSidebar.js
Comment (floreks) [reply www/src/components/shell/TerminalSidebar.js:557]: Good catch :slightly_smiling_face:

File: www/src/components/shell/TerminalSidebar.js
Comment (floreks) [reply www/src/components/shell/TerminalSidebar.js:506]: Oh yeah. Makes sense :slightly_smiling_face:

**Comment by floreks** [comment]
> Should we default to opening the Commands tab in the cheatsheet for now since there's only one tab?

I guess it would be best. There's no point in showing only a single option for now.

**Comment by floreks** [comment]
All done. PTAL @pluralsh/frontend

**Review by maciaszczykm — APPROVED** [review]
Looks good to me. Let's wait for @dogmar final approval.

**Review by dogmar — APPROVED** [review]

**Pull request merged**
